src/Service.Tests/dab-config.MsSql.json

```diff
@@ -2836,6 +2836,32 @@
         }
       }
     },
+    "DefaultBuiltInFunction": {
+      "source": {
+        "object": "default_with_function_table",
+        "type": "table"
+      },
+      "graphql": {
+        "enabled": true
+      },
```

Comment:
Would your added test for this change still pass if default values were defined here in the runtime config? If so, I think it is necessary to have an additional test within `MutationBuilderTests` to validate the schema generated for this entity: validate the expected presence (or absence) of a default value on the mutation defintion

Reply:
we don't support default values for tables or views in the runtime config yet.